packages/slate-editor/src/extensions/image/ImageExtension.tsx

```diff
@@ -159,3 +154,14 @@ export const ImageExtension = ({
     serialize: (nodes) => withoutNodes(nodes, isImageCandidateElement),
     withOverrides: withImages,
 });
+
+function replaceImageWithParagraph(editor: Editor, nodeEntry: NodeEntry<Node>) {
+    Transforms.setNodes<ImageNode | ParagraphNode>(
+        editor,
+        createParagraph({ align: Alignment.LEFT }),
```

Comment:
I think it'd be better to remove the `align` prop, instead of forcing it to `left`.

The reason is that, if omitted, paragraphs will inherit editor global `align` prop alignment (which may be different from `left`; i.e. RTL locales have it set to `right`).

So:

```tsx
Transforms.setNodes<ImageNode | ParagraphNode>(editor, createParagraph(), {
    at: nodeEntry[1],
    match: isImageNode,
});
Transforms.unsetNodes<ParagraphNode>(editor, 'align', {
    at: nodeEntry[1], 
    match: isParagraphNode,
});
```